shell/common/node_bindings.cc

```diff
@@ -132,6 +137,15 @@ bool IsPackagedApp() {
 #endif
 }
 
+void FatalErrorCallback(const char* location, const char* message) {
+  LOG(ERROR) << "Fatal error in V8: " << location << " " << message;
+
+  electron::crash_keys::SetCrashKey("electron.v8-fatal.message", message);
+  electron::crash_keys::SetCrashKey("electron.v8-fatal.location", location);
+
+  electron::ElectronBindings::Crash();
```

Comment:
This is kinda a /nit and the compiler might do it for us anyway, but can we rename this method `V8FatalErrorCallback` and actually do the crash inline here?  That way the stacktrace when it is reported ends with `V8FatalErrorCallback` instead of `ElectronBindings::Crash` which isn't super helpful